src/AdobeKit-dev.js

```diff
@@ -214,7 +214,10 @@
             appMeasurement.linkTrackEvents = appMeasurement.events || null;
             processProductsAndSetEvents(event, linkTrackVars);
             appMeasurement.pageName = event.EventName || window.document.title;
-            linkTrackVars.push('products', 'events', 'pageName');
+            linkTrackVars.push('products', 'events');
+            if (settings.enablePageName) {
```

Comment:
I think should be checking for `'True'`

Reply:
ah good catch. i only cherrypicked the first commit from here when rebasing due to conflicts https://github.com/mparticle-integrations/mparticle-javascript-integration-adobe/pull/6/commits